dev_guide/managing_images.adoc

```diff
@@ -777,3 +777,68 @@ NAME       DOCKER REPO                        TAGS      UPDATED
 my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 ====
+[[writing-image-streams-for-s2i-builders]]
+== Writing Image Streams for S2I Builders
```

Comment:
Oh, the main reason I was looking at this was because it was breaking the formatting in the Customer Portal version, since the section title was too close to the code block above it.